index.html

```diff
@@ -56,6 +56,7 @@ <h2>Edit your signature</h3>
       <td>
         <select id="countrySelector">
           <option value="France">France</option>
+          <option value="Lyon">France - Lyon</option>
           <option value="UK">UK</option>
           <option value="US">US</option>
         </select>
```

Comment:
I'm not sure that the right way to do it. I could also add a new "Office" field display when 'France' is picked to select between 'Paris' and 'Lyon'.